test/WebJobs.Script.Tests/Description/FunctionInvokerBaseTests.cs

```diff
@@ -35,33 +59,131 @@ public void LogInvocationMetrics_EmitsExpectedEvents()
             Assert.Equal("function.binding.table.In", metrics.LoggedEvents[5]);
         }
 
+        [Fact]
+        public async Task Invoke_Success_EmitsExpectedEvents()
+        {
+            var executionContext = new ExecutionContext
+            {
+                InvocationId = Guid.NewGuid()
+            };
+            var parameters = new object[] { "Test", _traceWriter, executionContext };
+            await _invoker.Invoke(parameters);
+
+            Assert.Equal(1, _metricsLogger.MetricEventsBegan.Count);
+            Assert.Equal(1, _metricsLogger.EventsBegan.Count);
+            Assert.Equal(1, _metricsLogger.MetricEventsEnded.Count);
+            Assert.Equal(1, _metricsLogger.EventsEnded.Count);
+
+            // verify started event
+            var startedEvent = (FunctionStartedEvent)_metricsLogger.MetricEventsBegan[0];
```

Comment:
Also adding missing test coverage for this important event on both success and error cases